dlink-admin/src/main/java/com/dlink/model/Cluster.java

```diff
@@ -76,4 +76,21 @@ public static Cluster autoRegistersCluster(String hosts, String name, String ali
         cluster.setEnabled(true);
         return cluster;
     }
+    public String getJobManagerHost() {
+        if (type.equals("yarn-session")) {
+            if (StrUtil.isBlank(resourceManagerAddr) || StrUtil.isBlank(applicationId))
+                return jobManagerHost;
```

Comment:
Where is resourceManagerAddr defined?
Where is applicationId defined?

Reply:
Yarn session's config StrUtil.isBlank(resourceManagerAddr) and  StrUtil.isBlank(applicationId) was satisfied  , will manage.
the old Yarn session  jobManagerHost config, dont manged ,that's Previous way.

resourceManagerAddr  applicationId in db columns